engine/src/main/java/io/camunda/zeebe/engine/state/deployment/DbDecisionState.java

```diff
@@ -161,8 +161,7 @@ public List<PersistedDecision> findDecisionsByDecisionRequirementsKey(
     return decisions;
   }
 
-  @Override
-  public Optional<Long> findPreviousVersionDecisionKey(
+  private Optional<Long> findPreviousVersionDecisionKey(
```

Comment:
Now that the method is private, we lost the JavaDoc which may have been useful.

🔧 If you think it makes sense, feel free to re-add the JavaDoc